### PR TITLE
Exams by Anton Dovzhenko, IM-22 group

### PR DESCRIPTION
**Pull request by KobbAsa**

### CODE REVIEW

**Comment by M-Rybalko** [comment]
Good job, code indeed became better

File: .eslintrc.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     ],
     "linebreak-style": [
       "error",
-      "unix"
+      "windows"
```
Comment (tshemsedinov) [inline .eslintrc.json:21]: https://youtu.be/W87wOCSPA08

**Comment by tshemsedinov** [comment]
Do not commit a bunch of IDE and environment files, just sources

File: Tasks/day.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
 // Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```
Comment (tshemsedinov) [inline Tasks/day.js:4]: `DAYS`

File: Tasks/password.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-let GeneratePassword = (alphabet, length) => {
-  const MAX = alphabet.length;
-  key = '';
+const generatePassword = (alphabet, length) => {
+  let key = '';
   for (let i = 0; i < length; i++) {
-    Index = Math.floor(Math.random() * MAX);
-    key = key + alphabet[Index];
+    const index = Math.floor(Math.random() * alphabet.length);
```
Comment (tshemsedinov) [inline Tasks/password.js:7]: I'd prefer to extract `random` as a separate function

**Pull request closed without merging**